### ISSUE TITLE
[a11y][theme] Text color does not meet WCAG standards for accessible text

### ISSUE THREAD

**Issue by staghouse** [vuejs/vitepress#2038]
### Describe the bug

Some design tokens (CSS Variables) for text color do not meet a11y standards for accessible text color. I am noticing particularly with `--vp-c-text-2` in the light theme. Its off by a delta of only ~0.2 points so I'm betting there is a happy medium of a minor adjustment that can be made to the colors to meet the standard of a 4.5:1 ratio.

### Reproduction

- Visit https://vitepress.vuejs.org/
- Enable light theme
- Use the browser inspect accessibility tab and filter by contrast (I use Firefox)
- Notice all the checks about contrast

### Expected behavior

There should be no contrast checks for light theme (also dark theme should be audited as well).

### System Info

```shell
System:
    OS: macOS 12.1
    CPU: (10) arm64 Apple M1 Pro
    Memory: 144.91 MB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.16.0 - /usr/local/bin/node
    Yarn: 1.22.19 - /opt/homebrew/bin/yarn
    npm: 8.19.2 - /opt/homebrew/bin/npm
  Browsers:
    Chrome: 110.0.5481.177
    Firefox: 110.0.1
    Safari: 15.2
  npmPackages:
    vitepress: 1.0.0-alpha.49 => 1.0.0-alpha.49
```

### Validations

- [X] Follow our [Code of Conduct](https://vuejs.org/about/coc.html)
- [X] Read the [docs](https://vitepress.vuejs.org).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

**Comment by yyx990803** [comment]
Fixed in 3b6a6d1 - `--vp-c-text-2` is now contrast compliant. The remaining contrast issues will have to do with the choice of brand color.
